kaniko.go

```diff
@@ -264,3 +283,60 @@ func getDigest(digestFile string) string {
 func trace(cmd *exec.Cmd) {
 	fmt.Fprintf(os.Stdout, "+ %s\n", strings.Join(cmd.Args, " "))
 }
+
+func getPluginEnvVars(cmdArgs []string) []string {
```

Comment:
Add a unit test which tests basic functionality - cases like adding a plugin env var when it's already been specified in `p.Build`, etc

Reply:
done